_chapters/02_category.md

```diff
@@ -423,62 +423,62 @@ There are some types of categories in which only one morphism between two object
 Composition
 ---
 
-The most important requirement for a structure to be called a category is that *two morphisms can make a third*, or in other words, that morphisms are *composable* --- given two successive arrows with appropriate type signature, we can draw a third one that is equivalent to the consecutive application of the other two.
+The most important requirement for a structure to be called a category is that *two morphisms can make a third*, or in other words, that morphisms are *composable* --- given two successive arrows with appropriate type signatures, we can draw a third one that is equivalent to the consecutive application of the first two functions.
 
 ![Composition of morphisms](../02_category/composition.svg)
 
-Formally, this requirement says that there should exist an operation (denoted with the symbol $•$) such that for each two functions $g: A → B$ and $f: B → C$, there exists a function $(f • g): A → C$ (again, note that there can be many other morphisms with with the same type signature, but there must be *exactly one* morphism that fits these criteria).
+Formally, this requirement says that there should exist an operation (denoted with the symbol $•$) such that for each of the two functions $g: A → B$ and $f: B → C$, there exists a function $(f • g): A → C$ (again, note that there can be many other morphisms with the same type signature, but there must be *exactly one* morphism that fits these criteria).
 
 ![Composition of morphisms in the context of additional morphism](../02_category/composition_arrows.svg)
 
-**NB:** Note (if you haven't already) that functional composition is written from right to left. e.g. applying $g$ and then applying $f$ is written $f • g$ and not the other way around. (You can think of it as a shortcut to $f(g(a))$).
+**NB:** Note (if you haven't already) that functional composition should be performed from right to left. e.g. applying $g$ and then applying $f$ is written $f • g$ and not the other way around. (You can think of it as a shortcut to $f(g(a))$).
```

Comment:
For me, "is written" is clearer, as it shows that we are talking about a purely notational thing, while when you say that "it should be performed from right to left" it's not clear what you mean